src/main/groovy/edu/oregonstate/mist/personsapi/PersonsResource.groovy

```diff
@@ -377,8 +377,22 @@ class PersonsResource extends Resource {
                 // when changeReasonCode become a required parameter.
                 job.changeReasonCode = null
             }
-            switch (employmentType) {
+            if (update) {
+                if (changeReasonCode == 'LCHNG') {
+                    logger.info("Labor change. Updating $employmentType job")
+                    dbFunctionOutput = bannerPersonsWriteDAO.updateLaborChangeJob(osuID, job)
+                        .getString(BannerPersonsWriteDAO.outParameter)
+                }
+                // TODO: SP hasn't be released yet. Need to test this part once it's done.
+                // else {
+                //     logger.info("Updating $employmentType job")
+                //     dbFunctionOutput = bannerPersonsWriteDAO.updateJob(osuID, job)
+                //         .getString(BannerPersonsWriteDAO.outParameter)
+                // }
+            } else {
+                switch (employmentType) {
                 case studentEmploymentType:
+                    // TODO: This must be removed once the updateJob SP is completed.
                     if (update) {
```

Comment:
I see the comment saying this will be removed but I don't think its reachable now anyways. If `update` is true then we won't be in this `else`

Reply:
Ah, good catch, will fix it.